src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs

```diff
@@ -21,7 +21,8 @@ public record ExperimentalFeaturesEnabled(
     bool SecureOutputs,
     bool ResourceInfoCodegen,
     bool TypedVariables,
-    bool ModuleExtensionConfigs)
+    bool ModuleExtensionConfigs,
+    bool ResourceDSC)
```

Comment:
[nit] The convention we follow in this codebase is for camelcase/pascal case to also apply to acronyms (e.g. `ResourceDsc` rather than `ResourceDSC`)

Reply:
I wondered! Can do.